lib/models/Snacks.js

```diff
@@ -0,0 +1,58 @@
+const pool = require('../utils/pool');
+
+module.exports = class Snacks {
+  id;
+  snackname;
+  snacktime;
+
+  constructor(row) {
+    this.id = row.id;
+    this.snackname = row.snackname;
+    this.snacktime = row.snacktime;
```

Comment:
I know this was from a few days/labs ago and we just went over this today in Lab BUT.....remember the left side of the constructor is your JavaScript land and the right side is what you're converting it to be in SQL-land. 

JavaScript is camelCase whereas SQL is snake_case, so you want to always make sure these are updated appropriately and not the exact same.